functions/avg.yml

```diff
@@ -15,14 +15,14 @@ desc: |
   An empty array will produce a return value of null.
 examples:
 - context: [10, 15, 20]
```

Comment:
Have you considered naming this property `given` instead of `context` like in the original site ?

Reply:
I think "context" makes more sense, especially when we want to start describing expressions within a `let()` context.